operator/option/config.go

```diff
@@ -30,6 +30,10 @@ const (
 	// SkipTracingPolicyCRD specifies whether the tracing-policies CustomResourceDefinition will be
 	// disabled
 	SkipTracingPolicyCRD = "skip-tracing-policy-crd"
+
+	// ForceUpdateCRD specifies whether operator should ignore current CRD version
+	// and forcefully update it.
+	ForceUpdateCRD = "force-update-crd"
 )
```

Comment:
Again, plura makes more sense to mere here. That is, `force-update-crds`.

Reply:
Updated in operator and chart